certificate-policy.md

```diff
@@ -1250,26 +1250,24 @@ No stipulation.
 
 #### 6.5.1 Specific computer security technical requirements
 
-Administrator privileges to all Certificate System Components shall only be granted to the Administrator trusted role.  All CAs shall implement multi-factor or multi-party authentication for all Administrator trusted role access to Certificate System Components including operating system and software.
```

Comment:
@debcooley why remove these explicit statements for multi-factor / multi-party authentication?  Is it because it's **assumed** as a default USG requirement? 

The requirement statements were mapped from the network system security requirements for CA/B.